pyhht/visualization.py

```diff
@@ -72,7 +73,8 @@ def plot_imfs(signal, imfs, time_samples=None, fignum=None, show=True):
     ax.tick_params(which='both', left=False, bottom=False, labelleft=False,
                    labelbottom=False)
     ax.grid(False)
-    ax.set_ylabel('Signal')
+    ax.set_ylabel('Signal',rotation=0)
+    ax.yaxis.set_label_coords(-0.1,0)
```

Comment:
These parameters should also be exposed in the `plot_imfs` function, with these values as default.